doc/build-osx.md

```diff
@@ -96,11 +96,13 @@ Skip if you don't intend to use the GUI.
 brew install qt@5
 ```
 
-Ensure that the `qt@5` package is installed, not the `qt` package.
-If 'qt' is installed, the build process will fail.
-if installed, remove the `qt` package with the following command:
+_Note:_ Ensure that _only_ `qt@5` is installed:
```

Comment:
**nit**: This seems to be the only place where "Note" is italicized in the doc; it might be worth reverting to match the other notes. Italicizing also seems redundant here for emphasis, as it's already a note. 

```suggestion
Note: Ensure that _only_ `qt@5` is installed:
```

Reply:
Done :)